src/control/lib/control/auto.go

```diff
@@ -27,7 +27,7 @@ const (
 	defaultFiPort         = 31416
 	defaultFiPortInterval = 1000
 	defaultTargetCount    = 16
-	defaultIOSrvLogFile   = "/tmp/daos_engine"
+	defaultEngineLogFile   = "/tmp/daos_engine"
```

Comment:
we are missing a .log suffix?

Reply:
this is only the prefix, the suffix and extension get added later, I will update the variable name in a separate PR